### ISSUE TITLE
False positive with slash continuation lines

### ISSUE THREAD

**Issue by peterjc** [keisheiled/flake8-implicit-str-concat#4]
e.g. Call this file ``sql.py``

```python
"""Small Python sample to test flake8 plugins."""

sql = 'SELECT seqfeature_id, type.name, "rank"' \
    " FROM seqfeature join term type on (type_term_id = type.term_id)" \
    " WHERE bioentry_id = %s" \
    ' ORDER BY "rank"'

print(sql)
```

Running flake8:

```
$ flake8 sql.py
sql.py:3:7: BLK100 Black would make changes.
sql.py:3:48: implicit-str-concat implicitly concatenated string literals on one line
sql.py:4:71: implicit-str-concat implicitly concatenated string literals on one line
sql.py:5:30: implicit-str-concat implicitly concatenated string literals on one line
```

Issue #3 covers the lack of a plugin prefix, e.g. call this ``ISC001``.

The ``BLK100`` warning is from my own plugin https://github.com/peterjc/flake8-black - black would reformat the file as follows:

```python
"""Small Python sample to test flake8 plugins."""

sql = (
    'SELECT seqfeature_id, type.name, "rank"'
    " FROM seqfeature join term type on (type_term_id = type.term_id)"
    " WHERE bioentry_id = %s"
    ' ORDER BY "rank"'
)

print(sql)
```

Nevertheless, your plugin would ideally not be triggered by this construct - although perhaps this could trigger a different error (maybe ``ISC002``)?

**Comment by keisheiled** [comment]
You think this will do it? I didn't consider continuation lines 

```python
    return (
        (
            *a.end,
            (
                "ISC002 implicitly concatenated string literals over continuation line"
                if a.end[0] == b.start[0]
                else "ISC001 implicitly concatenated string literals on one line"
            )
            None,
        )
        for (a, b) in pairwise(file_tokens)
        if a.type == b.type == tokenize.STRING
    )
```

**Comment by peterjc** [comment]
Missing a comma before the ``None``, but that doesn't seem to trigger ``ISC001`` on my example...

**Comment by keisheiled** [comment]
@peterjc how about this?

```python
    return (
        (
            *a.end,
            (
                "implicit-str-concat-continuation implicitly concatenated string literals over continuation line"
                if a.end[0] == b.start[0]
                else "implicit-str-concat-oneline implicitly concatenated string literals on one line"
            ),
            None,
        )
        for (a, b) in pairwise(file_tokens)
        if a.type == b.type == tokenize.STRING
    )
```

**Comment by peterjc** [comment]
That works - but I can't immediately see why the exact message string matters.

P.S. You can remove the brackets round the if/else

**Comment by keisheiled** [comment]
@peterjc https://github.com/keisheiled/flake8-implicit-str-concat/pull/5 it's because I only claim errors codes with the prefix `implicit-str-concat`

**Comment by peterjc** [comment]
Ah - I see it now in ``pyproject.toml``, I hadn't looked there. Yes, that would need changing to ``ISC`` if you were to use ``ISC001`` etc.

I've answered on #5, but I would strongly prefer you adopted a three letter prefix like ``ISC`` and the three digit convention.
